generate

```diff
@@ -158,7 +158,7 @@ def recurse(aspect, level=0):
                 output += "* `" + subsub_name + " <" + subsub_name + "/README.rst>`_\n"
         else:
             output += "This aspect does not have any sub aspects.\n\n"
-
+        output += "\n"
```

Comment:
Also, please run the `generate` file to generate the latest version so that we can verify if it works perfectly.